isis/appdata/serialnumbers/vikSerialNumber0003.trn

```diff
@@ -0,0 +1,33 @@
+Group = Keyword1
```

Comment:
Upper camel case and should it be renamed to contain the full mission name?

Reply:
done in new PR